### ISSUE TITLE
Please update npm dependency

### ISSUE THREAD

**Issue by superlbr** [Automattic/wp-calypso#4344]
old npm versions, old babel and so many deprecated npms,I just wasted half a day installing npm in centos 7.2

**Comment by aduth** [comment]
Hi @superlbr !

- Our reasoning for using npm v2.x (instead of 3.x) is because it aligns with our [current production version of Node](https://github.com/Automattic/wp-calypso/blob/fd9685a5ba656061d28aa7534dfeda80e548dfe6/Dockerfile#L17), 4.3.0. We use Node 4.x because this is the [latest LTS release available](https://nodejs.org/en/download/releases/), though (a) we're a minor point version behind now and (b) we've recently considered adopting a newer version of Node despite lack of LTS designation.
 - Node releases: https://nodejs.org/en/download/releases/
 - Node LTS strategy: https://github.com/nodejs/LTS
- We'd started the [upgrade process for Babel 6](https://github.com/Automattic/wp-calypso/pull/1515), but between performance regressions and general bugginess of Babel 6, we'd punted the upgrade as of January. Recent changes to our own build process for improving performance (e.g. #4154) may warrant another look at Babel build times.

In summary, yes, we should at least take another look at both of these points for upgrade :smiley:

**Comment by nb** [comment]
We upgraded in #4694 🎉 .

Thanks for the request, @superlbr.
